yarn.lock

```diff
@@ -9677,12 +9684,13 @@ truffle-assertions@^0.9.1:
     assertion-error "^1.1.0"
     lodash.isequal "^4.5.0"
 
-truffle-flatten@^1.0.5:
-  version "1.0.5"
-  resolved "https://registry.yarnpkg.com/truffle-flatten/-/truffle-flatten-1.0.5.tgz#0d39a73ae689b78cec8764bf09a9fe0dbd35f1ca"
-  integrity sha512-oGCmrj0h/Dpr5K2TDOEb7BQU5NG8pgDv0VLmPTL2+Il8Cd+BzgaT6E7+NpM52YeyCvPutYp5sAaEetDJqN1n/w==
+truffle-flatten@^1.0.6:
```

Comment:
Do you mind reverting the yarn.lock to how it was before please? This includes an update to 2 truffle tools we use and we should assess whether we want the update on those before it goes in.

Reply:
it was updated 11 days ago in package.json to 1.0.6, but the lockfile was never bumped. they should be reconciled

https://github.com/airswap/airswap-protocols/commit/2694f0a44400e5e6eaf723cce7a489240c22d200

Reply:
Ah good spot hadn't noticed that.

Reply:
if we want to use 1.0.6, yarn.lock should be bumped since it's gonna get bumped on any machine that runs `yarn` in the future, and I think also in CI.

if don't want to use 1.0.6, we should revert the commit linked above. 


But, it shouldn't stay as is since it's a mismatch in the current state.

Let me know what you think and I can update @alicevhenshaw @poiresel